src/azure-cli/azure/cli/command_modules/container/tests/latest/recordings/test_container_create_with_acr.yaml

```diff
@@ -45,7 +45,7 @@ interactions:
     body: '{"properties": {"containers": [{"name": "clicontainer000002", "properties":
       {"image": "clitestregistry1.azurecr.io/nginx:latest", "resources": {"requests":
       {"memoryInGB": 1.5, "cpu": 1.0}}}}], "imageRegistryCredentials": [{"server":
-      "clitestregistry1.azurecr.io", "username": "clitestregistry1", "password": "qtsA=TxYOyZ8kkhFXWHe7mQbFN7fxiLj"}],
+      "clitestregistry1.azurecr.io", "username": "clitestregistry1", "password": "passplaceholder"}],
```

Comment:
I believe this secret should be the password for the deleted test resource, so it does not pose a security issue. @joseph-porter Please help confirm it~